### ISSUE TITLE
Inter Mutuelles Assistance GIE (dont la MACIF)

### ISSUE THREAD

**Issue by Rpnpif** [osm-fr/attributions#467]
### Ce signalement concerne…

- [ https://suivi.ima.eu/](https://suivi.ima.eu/)  a une carte en ligne issue d'OpenStreetmap.. C'est une page dont l'accès est réservé aux clients.

### Présence de l'attribution

❌ Pas d'attribution

### Prise de contact

❌ Je ne l'ai pas contacté

### Informations supplémentaires

- Responsable contacté le : 
  - Responsable contacté par (courriel, Twitter…) : 
- Lié au signalement : #

**Comment by Jean-BaptisteC** [comment]
Est-il possible d'avoir une capture d'écran du fond de plan ?

**Comment by Rpnpif** [comment]
Oui. Il s'agit de l'adresse https://suivi.ima.eu/main/case?token=<ICI_LE_TOKEN_EN_QUESTION>.
![Screenshot 2024-08-02 at 09-46-24 Suivi d'assistance](https://github.com/user-attachments/assets/ff0d967b-9116-4840-84ae-55a6b8ceb89f)

**Comment by vinber** [comment]
Je confirme par l'expérience... Pas d'attribution.

**Comment by Jean-BaptisteC** [comment]
Le site semble a voir changé, a vérifier sur le nouveau

**Comment by NathanBnm** [comment]
@Rpnpif @vinber est-ce toujours le cas aujourd'hui ?

**Comment by Rpnpif** [comment]
Je n'ai malheureusement plus accès à cette page.
